### DISCUSSION TITLE
Need help installing Velero with NFS/local as a backup storage

### DISCUSSION THREAD

**Discussion by lailahamza** [vmware-tanzu/velero#6166] [Community support Q&A]
Hello,

I hope this email finds you all doing well. I am writing to seek your help in installing Velero with NFS or local as backup storage. I have been trying to do it myself, but unfortunately, my attempts have been unsuccessful so far.

I have followed the documentation and searched for solutions online, but I haven't been able to get it working properly. I believe that the problem lies with configuring the NFS backup storage, and I am hoping that someone in the group who has experience with this can help me out.

If any of you have successfully installed Velero with NFS as backup storage, I would greatly appreciate your assistance. I am willing to schedule a call or a meeting to discuss the installation process and troubleshoot any issues together.

Thank you in advance for your help. I am looking forward to hearing from you soon.

Best regards,
Laila HZ

**Comment by sseago** [comment]
NFS is not currently supported for backup storage. Backup storage is handled via Velero object store plugins. Currently there are three supported Object Store Plugins: s3/AWS, Azure, and GCP. There have been discussions about eventually implementing an NFS and/or filesystem object store plugin, but there is no implementation of this as of now.

**Reply by farazkhawaja** [reply]
@sseago any update on this?

**Reply by kaovilai** [reply]
No updates. You can try https://github.com/replicatedhq/local-volume-provider in the meantime.

**Comment by Lyndon-Li** [comment]
Please vote on issue #1229 and share your use case of why NFS is required over minio
